app/views/course/forum/topics/_controls.html.slim

```diff
@@ -0,0 +1,39 @@
+span.btn-group
+  - if @topic.subscribed_by?(current_user)
+    = link_to unsubscribe_course_forum_topic_path(current_course, @forum, @topic),
+              title: t('course.forum.topics.unsubscribe.tag'), class: ['btn', 'btn-danger'],
```

Comment:
actually if there is a problem with button_to, you can declare another style, see http://stackoverflow.com/questions/20821171/bootstrap-button-group-styling-with-button-to